examples/viewer.py

```diff
@@ -733,6 +783,51 @@ def exit_event(self, event: Application.ExitEvent):
         event.accepted = True
         exit(0)
 
+    def draw_text(self, sensor_subtype):
+        self._shader.bind_vector_texture(self._cache.texture)
+        self._shader.transformation_projection_matrix = (
+            self._transformation_projection_window_text
+        )
+        self._shader.color = [1.0, 1.0, 1.0]
+        self._shader.outline_color = [1.0, 1.0, 1.0]
+        self._shader.outline_range = (0.45, 0.40)
+
+        sensor_subtype_string = self.get_sensor_subtype_string(sensor_subtype)
+        if self.mouse_interaction == MouseMode.LOOK:
+            mouse_mode_string = "LOOK"
+        elif self.mouse_interaction == MouseMode.GRAB:
+            mouse_mode_string = "GRAB"
+        placeholder = "Oprah"
+        self._window_text.render(
+            f"""
+{self.fps} FPS
+{placeholder} drawables, {placeholder} culled
+{sensor_subtype_string}
+Last {self.num_frames_to_track} frames:
+    Frame time: {placeholder} ms
+    CPU duration: {placeholder} ms
+    GPU duration: {placeholder} µs
+    Vertex fetch ratio: {placeholder}
+    Primitives clipped: {placeholder} %
```

Comment:
Ah yes, I should expose the FrameProfiler to Python as well. Adding a TODO for myself.

Reply:
Ah thank you, I was looking into the gl module in Python to see if there were any profiling options, but just decided to bring it up later with the team